### ISSUE TITLE
Prototype pollution low severity vulnerability by merge-options package

### ISSUE THREAD

**Issue by losbeekos** [JetBrains/svg-mixer#42]
I've run an `npm audit` on my packages and one of my packages uses svg-baker which throws these 4 low severity vulnerabilities, more info can be found here: https://nodesecurity.io/advisories/717. I think we could find an alternative for the merge-options or maybe notify the merge-options author. What do you guys think?

NPM audit log:
```
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ svg-sprite-loader                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ svg-sprite-loader > svg-baker > merge-options                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/717                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ svg-sprite-loader                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ svg-sprite-loader > svg-baker > posthtml-svg-mode >          │
│               │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/717                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ svg-sprite-loader                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ svg-sprite-loader > svg-baker-runtime > svg-baker >          │
│               │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/717                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ merge-options                                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ svg-sprite-loader                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ svg-sprite-loader > svg-baker-runtime > svg-baker >          │
│               │ posthtml-svg-mode > merge-options                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/717                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by vtereshyn** [comment]
Any updates?

**Comment by kisenka** [comment]
Fixed in svg-sprite-loader@4.1.3, please update

**Comment by losbeekos** [comment]
Thanks for the update!
